packages/ethereum-storage/src/ethereum-utils.ts

```diff
@@ -1,8 +1,7 @@
-import { CurrencyTypes, StorageTypes } from '@requestnetwork/types';
+import { StorageTypes } from '@requestnetwork/types';
 import * as config from './config';
 
 import { BigNumber } from 'ethers';
-import { EvmChains } from '@requestnetwork/currency';
```

Comment:
@alexandre-abrioux FYI. 
I'm trying to reduce inter-packages dependencies... The currency package doesn't make sense in the storage I think